test/integration/tasks.test.js

```diff
@@ -80,6 +81,46 @@ describe("Tasks", function () {
     sinon.restore();
   });
 
+  describe("GET /tasks/overdue/current", function () {
+    it("Should return all the current overdue Tasks", function (done) {
+      chai
+        .request(app)
+        .get("/tasks/overdue/current")
+        .set("cookie", `${cookieName}=${superUserJwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body.message).to.be.equal("Overdue Tasks returned successfully!");
+          expect(res.body.overdueTasks[0].id).to.be.oneOf([taskId, taskId1]);
+          expect(res.body.overdueTasks[1].id).to.be.oneOf([taskId, taskId1]);
```

Comment:
can we check their ends on parameter to be more sure i.e. the ends on is less than the current date

Reply:
that is we can be sure of, as there is already a check for that in the code.

Reply:
tests help us know if the code written to handle this works fine.